onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -762,6 +766,10 @@ def _where_input_wrangler(
         reason="fixme: ORT fails with invalid model: 'INVALID_ARGUMENT : Failed to load model with error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)'",
         test_class_name="TestOutputConsistencyFullGraph",
     ),
+    xfail(
+        "vstack",
+        reason="fixme: A bug of constant-propagation optimization within the subgraph, we can avoid it by turning off graph-optimizations in session options",
```

Comment:
Should we turn it off in test and file a bug to ORT?

Reply:
Would nested affect performance?

Reply:
xfail actually works. I think CI breaks because of something else. Actually @gramalingam spot the bug, but I can still file an issue to track.

Reply:
https://github.com/microsoft/onnxruntime/issues/16296